src/Microsoft.Sbom.Api/Executors/Spdx30SerializationStrategy.cs

```diff
@@ -15,81 +14,97 @@ namespace Microsoft.Sbom.Api.Workflows.Helpers;
 /// </summary>
 internal class Spdx30SerializationStrategy : IJsonSerializationStrategy
 {
+    /// <summary>
+    /// inheritdoc
+    /// </summary>
+    /// <param name="elementsSupportingConfigs"></param>
+    /// <param name="config"></param>
+    /// <returns>Always returns false since we do not want to write a separate files array in SPDX 3.0.
+    /// A separate files array is only supported for SPDX 2.2.</returns>
     public bool AddToFilesSupportingConfig(IList<ISbomConfig> elementsSupportingConfigs, ISbomConfig config)
     {
         elementsSupportingConfigs.Add(config);
-        return true;
+        return false;
     }
 
+    /// <summary>
+    /// inheritdoc
+    /// </summary>
+    /// <param name="elementsSupportingConfigs"></param>
+    /// <param name="config"></param>
+    /// <returns>Always returns false since we do not want to write a separate packages array in SPDX 3.0.
+    /// A separate packages array is only supported for SPDX 2.2.</returns>
     public bool AddToPackagesSupportingConfig(IList<ISbomConfig> elementsSupportingConfigs, ISbomConfig config)
     {
         elementsSupportingConfigs.Add(config);
-        return true;
+        return false;
     }
 
+    /// <summary>
+    /// inheritdoc
+    /// </summary>
+    /// <param name="elementsSupportingConfigs"></param>
+    /// <param name="config"></param>
+    /// <returns>Always returns false since we do not want to write a separate relationships array in SPDX 3.0.
+    /// A separate relationships array is only supported for SPDX 2.2.</returns>
     public bool AddToRelationshipsSupportingConfig(IList<ISbomConfig> elementsSupportingConfigs, ISbomConfig config)
     {
-        return true;
+        return false;
     }
 
+    /// <summary>
+    /// inheritdoc
+    /// </summary>
+    /// <param name="elementsSupportingConfigs"></param>
+    /// <param name="config"></param>
+    /// <returns>Always returns false since we do not want to write a separate external document references array in SPDX 3.0.
```

Comment:
```suggestion
    /// <returns>Always returns false since we do not want to write a separate external document references array in SPDX 3.0.</returns>
```